static/docs/commands-reference/version.md

```diff
@@ -19,6 +19,7 @@ system/environment:
 | `DVC version`     | Version of DVC (along with a Git commit hash in case of a development version)                                                                            |
 | `Python version`  | Version of the Python being used for the project in which DVC is initialized                                                                              |
 | `Platform`        | Information about the operating system of the machine                                                                                                     |
+| `Binary`          | Shows whether the package is installed from a binary release or source                                                                                    |
```

Comment:
let's explain on example the difference between binary and source in our case - self-contained installer, homebrew, etc vs pip

is pip btw the only installation from sources?